### PR TITLE
Contribution Guidelines

### PR DESCRIPTION
**Pull request by LePips**
If anybody has any thoughts, leave a comment. I also document behaviors that are due to technical limitations so that they are surfaced to people.

### CODE REVIEW

**Review by PangMo5 — DISMISSED** [review]
Great docs XD

File: README.md [hunk 1]
```diff
@@ -32,17 +32,19 @@ Check out our [Weblate instance](https://translate.jellyfin.org/projects/swiftfi
 
 ## ⚙️ Development
 
-Xcode 13.0 with command line tools.
+Thank you for your interest in Swiftfin, please check out the [Contribution Guidelines](https://github.com/jellyfin/SwiftFin/contributing.md) to get started.
```
Comment (Shadowghost) [inline README.md:35]:
```suggestion
Thank you for your interest in Swiftfin, please check out the [Contribution Guidelines](https://github.com/jellyfin/Swiftfin/contributing.md) to get started.
```

**Comment by PangMo5** [comment]
In the Git Flow section,
Does it mean the flow used in Swiftfin?
else does it mean git-flow?

**Comment by LePips** [comment]
> In the Git Flow section, Does it mean the flow used in Swiftfin? else does it mean git-flow?

It would mean using git for PRs and such, which doesn't have an "official" naming scheme but commonly goes by git flow.

**Review by acvigue — APPROVED** [review]
looks good to me! 🎉

**Pull request merged**